blockchains/ethereum/assets/0xD1670Cb7c01987a8DdD6976C4894463460Eeb8a2/info.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "DMM: USDK",
+    "website": "https://defimoneymarket.com.com",
```

Comment:
Please change `https://defimoneymarket.com.com` to `https://defimoneymarket.com`